### ISSUE TITLE
import-outside-toplevel: unexpanded %s

### ISSUE THREAD

**Issue by jwilk** [PyCQA/pylint#3204]
### Steps to reproduce

Run Pylint against this code:
```python
def eggs():
    from . import ham
```

### Current behavior
```
test.py:2:4: C0415: Import outside toplevel (%s) (import-outside-toplevel)
```

### Expected behavior
It should print actual module name in place of `%s`.

### pylint --version output
```
pylint 2.4.3
astroid 2.3.0
Python 3.7.3 (default, Apr  3 2019, 05:39:12) 
[GCC 8.3.0]
```

**Comment by nickdrozd** [comment]
Is this a problem with the representation of that kind of import?

```
(Pdb) node
<ImportFrom l.2 at 0x7f98c1957828>
(Pdb) node.names
[('ham', None)]
(Pdb) node.modname
''
```

It looks the same if `.` is replaced with `..`.

**Comment by PCManticore** [comment]
This was already fixed by @nickdrozd, thank you both!

**Comment by nickdrozd** [comment]
@PCManticore I don't think this really is fixed. Now it shows:

`bug.py:2:4: C0415: Import outside toplevel (.ham) (import-outside-toplevel)`

It's better than it was, but does this really make sense? Maybe it's passable.

**Comment by PCManticore** [comment]
@nickdrozd That looks acceptable to me. Do you think it's confusing or that it should be the full import? if it's the latter, that would mean we'd have to use the full import path for other import statements as well.
